src/test/ensemble_tests.cpp

```diff
@@ -50,31 +52,20 @@ class EnsembleFlowTest : public TestWithTempDir {
         config.setNireq(NIREQ);
 
         // Prepare request
+        prepareRequest(bs1requestData, request, customPipelineInputName);
+        requestData = bs1requestData;
+    }
+
+    void prepareRequest(const std::vector<float>& requestData, PredictRequest& request, const std::string& customPipelineInputName) {
         tensorflow::TensorProto& proto = (*request.mutable_inputs())[customPipelineInputName];
         proto.set_dtype(tensorflow::DataType::DT_FLOAT);
-        requestData = bs1requestData;
         proto.mutable_tensor_content()->assign((char*)requestData.data(), requestData.size() * sizeof(float));
         proto.mutable_tensor_shape()->add_dim()->set_size(1);
         proto.mutable_tensor_shape()->add_dim()->set_size(DUMMY_MODEL_INPUT_SIZE);
     }
 
     void checkResponse(int seriesLength, int batchSize = 1) {
```

Comment:
Can't we move it entirely to test_utils?

Reply:
We decided to leave it as is since in ensemble_test we can call it with only 1 argument or 2 at most. Otherwise we would have to pass members as well which would be less readable & more error prone.